ingest/ingest_pipeline.py

```diff
@@ -207,12 +275,14 @@ def validate_arguments(parsed_args):
     Returns:
         None
     """
-    if parsed_args.matrix_file_type == 'mtx' and (parsed_args.gene_file == None
-                                                    or parsed_args.barcode_file == None):
-        raise ValueError(
-            ' Missing arguments: --gene-file and --barcode-file. Mtx files '
-            'must include .genes.tsv, and .barcodes.tsv files. See --help for '
-            'more information')
+    if 'matrix_file' in parsed_args:
```

Comment:
Having two `if` statements is unnecessary; the second `if` could simply be an `and`.

Reply:
It'd through an error when it checks for parsed_args.matrix_file_type if there was an 'and' here. It'd say something like parsed_args has no attribute type 'matrix_file_type'.

Reply:
I don't think it would.

If the proposed `and` were there and `'matrix_file' in parsed_args` evaluated to false, then the conditional expression would immediately return false without evaluating `parsed_args.matrix_file_type`.

Reply:
resolved in b4f5c8b